Plugins/Events/Events/DMActionEvents.cpp

```diff
@@ -234,14 +322,20 @@ int32_t DMActionEvents::HandleDMMessageHook(CNWSMessage *thisPtr, CNWSPlayer *pP
         }
         case MessageDungeonMasterMinor::Goto:
         {
-            event += "JUMP";
-            DefaultSignalEvent();
+            event += "GOTO";
```

Comment:
Why the name change?

Reply:
It's how the button for that event is called in the DM Chooser window,  the button called `Jump` is for the JumpTargetToPoint event.

Reply:
Let's hope no one was using the old event without the args... :)